src/fluree/db/peer/http_api.clj

```diff
@@ -827,7 +830,16 @@
                                                                                                          (if type (str "/" type))
                                                                                                          (if key (str "/" key))) host)
                                           db          (<? (fdb/db (:conn system) db-name))]
-                                      (<? (verify-auth db auth authority))))
+                                      (<? (verify-auth db auth authority)))
+
+                                    jwt
+                                    (let [jwt-auth (-> (pw-auth/fluree-auth-map (:conn system) jwt) :auth)
```

Comment:
This would be more idiomatic as `(:auth (pw-auth/fluree-auth-map (:conn system) jwt))` since there is only one outer fn call being threaded by `->`. Our soon-to-be-Sonarqube overlords will flag these. :)